lib/Base/Image.cpp

```diff
@@ -18,11 +18,10 @@
 #include "glow/Base/Tensor.h"
 #include "glow/Support/Support.h"
 
-using namespace glow;
-
-#if WITH_PNG
 #include <png.h>
```

Comment:
Hmm, doesn't this still need to be wrapped by `WITH_PNG`? If it's not installed the header wouldn't be found.

Reply:
I've changed the CMakeLists.txt for this library to only include Image.cpp if PNG_FOUND, so this entire file won't be included.  I think that makes sense, since there's probably nothing in here that's useful if you don't have libpng.

Reply:
Ah, got it. It seems like `WITH_PNG` isn't used at all after this PR then? Then what happens when you try to link Base to the `image-classifier`, wouldn't that throw an error since `readPngImageAndPreprocess()` won't be found? It seems like the `image-classifier` also shouldn't be built if `!PNG_FOUND`

Reply:
Ah yes, I should remove `WITH_PNG`.  Good catch.  It looks like nothing under `tools/loader` is built unless`PNG_FOUND`... which is probably too conservative since we actually use that for `text-translator` and `model-runner`.  But that's a separate issue :-)